programs/tests/playTestsMSYS.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/sh -e
```

Comment:
Does it sounds plausible to have the few necessary changes (like `diff`) inside run-time variables, in order to have a single script ?